Packs/FancyEmails/Integrations/FancyEmails/FancyEmails.py

```diff
@@ -0,0 +1,372 @@
+from datetime import datetime
+
+import demistomock as demisto  # noqa: F401
+import pytz
+from CommonServerPython import *  # noqa: F401
+
+PARAMS = demisto.params()
+BACKGROUND_COLOR = PARAMS.get('BackgroundColor', '#272727')
+FOREGROUND_COLOR = PARAMS.get('ForegroundColor', '#FFFFFF')
+BANNER_TEXT_COLOR = PARAMS.get('BannerTextColor', '#FFFFFF')
+BANNER_COLOR = PARAMS.get('BannerColor', '#F51212')
+CUSTOM_CSS = PARAMS.get('CustomCss', '')
+TIMEZONE = PARAMS.get('TimeZone')
+ALIGNLOGO = PARAMS.get('AlignLogo')
+BASELOGO = PARAMS.get('Base64Logo')
+LOGOHEIGHT = PARAMS.get('LogoHeight')
+LOGOWIDTH = PARAMS.get('LogoWidth')
```

Comment:
It is a better practice to handle this in the main, create a class and initiate it with these params, and then pass the class in the functions to get access to it.
Constants should be constants, meaning they should be known from the start of the run and not be changed (mainly- not be created by user input).